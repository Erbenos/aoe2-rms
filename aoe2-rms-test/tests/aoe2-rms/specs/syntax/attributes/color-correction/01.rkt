#lang aoe2-rms
; Desert-themed lighting

<TERRAIN-GENERATION>
(color-correction 'CC_DESERT)
