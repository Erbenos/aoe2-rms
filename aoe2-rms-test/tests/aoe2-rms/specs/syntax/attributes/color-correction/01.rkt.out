<TERRAIN_GENERATION>
color_correction CC_DESERT
